packages/vitest/src/utils/index.ts

```diff
@@ -129,3 +129,14 @@ export function getCallLastIndex(code: string) {
 }
 
 export { resolve as resolvePath }
+
+class AggregateErrorPonyfill extends Error {
+  errors: unknown[]
+  constructor(errors: Iterable<unknown>, message = '') {
+    super(message)
+    this.errors = [...errors]
+  }
+}
+
+// AggregateError is supported in Node.js 15.0.0+
+export const AggregateError = global.AggregateError || AggregateErrorPonyfill
```

Comment:
is is a ponyfill, if you have a fallback?

Reply:
I suppose it is.
https://github.com/sindresorhus/object-assign/blob/e3f8d936ae33b7b1e2fb66aa7023a6b3b90972e6/index.js#L65
https://github.com/sindresorhus/ponyfill

Reply:
your second link shows that ponyfills don’t use native apis 👀

Reply:
I don't understand what ponyfill is now....🤔 
Actually I do not have any preference here. So I'm ok with changing the name or implementation.

Reply:
My understanding is that a ponyfill is just an implementation of a feature you can use explicitly. And polyfill is a feature that it fallbacks to, when original feature is not available. So, ponyfill is more consistent